### ISSUE TITLE
Workaround `HttpClient` (??) to improve `dnu restore` warning

### ISSUE THREAD

**Issue by dougbu** [aspnet/dnx#2394]
With latest DNX (`1.0.0-beta7-12290`) and a wireless connection, I just got a fair few of the following meaningless warnings when executing `dnu restore --lock --parallel --quiet`. This output is not useful and should be removed or improved.
```
Warning: FindPackagesById: Microsoft.AspNet.Html.Abstractions
  Error while copying content to a stream.
Warning: FindPackagesById: System.Text.Encoding.Extensions
  Error while copying content to a stream.
Warning: FindPackagesById: Microsoft.Framework.BufferEntryCollection.Sources
  Error while copying content to a stream.
Warning: FindPackagesById: System.Collections.Concurrent
  Error while copying content to a stream.
```

TL;DR
According to [one forum thread](https://social.msdn.microsoft.com/Forums/windowsapps/en-US/ee685fb9-d2aa-44c0-bd17-ea615e838aa6/httpclient-only-works-when-using-httprequestmessage-explicitly?forum=winappswithcsharp), `HttpClient` sometimes throws (actually this isn't 100% clear) an `Exception` with the "Error while copying content to a stream" message. The key the original poster pinpointed was whether or not the base address ended with a trailing slash; good if it did, bad otherwise.

**Comment by glennc** [comment]
@ChengTian can you investigate this and tell me if there is anything obvious we should be doing?

**Comment by ChengTian** [comment]
We don't set any `HttpClient.BaseAddress` and always leave it empty. So looks like [the forum thread](https://social.msdn.microsoft.com/Forums/windowsapps/en-US/ee685fb9-d2aa-44c0-bd17-ea615e838aa6/httpclient-only-works-when-using-httprequestmessage-explicitly?forum=winappswithcsharp) is irrelevant.

I also tried intentionally setting `HttpClient.BaseAddress` to URIs without trailing slash but still couldn't repro.

@glennc , looks like this is a very rare issue and we don't have repro.

**Comment by glennc** [comment]
Closing until we can consistent repro
